### DISCUSSION TITLE
How can I get my wallet back?

### DISCUSSION THREAD

**Discussion by Lemarc-Z** [DimensionDev/Maskbook#2441] [General]
Hi,

I am stupid enough to delete the google extension while I have not backup the wallet p-key..

What can I do now ?  Plz help me.

Thanks

**Comment by Jack-Works** [comment]
```
C:\Users\<UserName>\AppData\Local\Google\Chrome\User Data\Default\IndexedDB\chrome-extension_jkoeaghipilijlahjplgbfiocjhldnap_0.indexeddb.leveldb
```
does it exist?

**Reply by Lemarc-Z** [reply]
there are some similar files there. How do I use them?

**Reply by Jack-Works** [reply]
the extension ID must match otherwise it is containing files of other extensions.

**Reply by Jack-Works** [reply]
Our extension ID is:jkoeaghipilijlahjplgbfiocjhldnap

**Reply by Lemarc-Z** [reply]
Yes, I find the folder with LOG, MANIFEST-000001, 000004.log,000005.ldb,LOG.old, CURRENT, LOCK.    But since i've already signed up with another account, those files were already updated.  And I tried to recover the hard-disk, but did not find anything like these.

**Reply by Jack-Works** [reply]
That's very unfortunate, I guess it's highly likely to be lost. Maybe you should try to find some tool to open those `.ldb` files and see if your key still in it.

If hard-disk recovering cannot find the old version of those files, there is nothing we can help

**Reply by Lemarc-Z** [reply]
Thanks. I think I might find the .ldb back via hard-disk recovery. But how can I open it .?   it is encrypted.   or should I try this : https://javascript.info/indexeddb ?
